# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserBridgeParent.cpp
# Commit: fbdc7fd9cb96
# Full Hash: fbdc7fd9cb96f26850f979b6f23b13f7bbaf727c
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-07-25 09:40:10
# Regressor Bug: 1652144
# File Overlap Count: 1
# Description:
#   Bug 1652144 - Part 2: Consider current BrowsingContextGroup for process selection, r=farre
#   
#   This requires keeping track of the current process used to host documents with a
#   particular remote type loaded in each BrowsingContextGroup. Due to lifecycle
#   oddities, this set is kept separate from the existing subscribers set on
# ==============================================================================

diff -r 920a06a52b8b -r fbdc7fd9cb96 dom/ipc/BrowserBridgeParent.cpp
--- a/dom/ipc/BrowserBridgeParent.cpp	Fri Jul 24 20:24:33 2020 +0000
+++ b/dom/ipc/BrowserBridgeParent.cpp	Fri Jul 24 20:24:36 2020 +0000
@@ -58,7 +58,7 @@
 
   // Ensure that our content process is subscribed to our newly created
   // BrowsingContextGroup.
-  browsingContext->Group()->EnsureSubscribed(aContentParent);
+  browsingContext->Group()->EnsureHostProcess(aContentParent);
   browsingContext->SetOwnerProcessId(aContentParent->ChildID());
 
   // Construct the BrowserParent object for our subframe.